packages/starknet_provider/lib/src/model/declare_transaction.dart

```diff
@@ -49,7 +57,7 @@ class DeclareTransactionV2
   const factory DeclareTransactionV2({
     @Default('DECLARE') String type,
     @Default('0x2') String version,
-    required Felt max_fee,
+    required String max_fee, // As String because devnet only supports 16 bytes and not a Felt
```

Comment:
Ok, fixed, we change this field to String because devnet only support a 16 bytes length for this field ,so we can't send this field as felt (too long)